code.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "name": "knoten",
+    "organization": "U.S. Geological Survey",
+    "description": "GitHub code repository for the knoten package",
+    "version": "0.2.1",
+    "status": "Pending",
+
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://github.com/USGS-Astrogeology/knoten/blob/master/LICENSE.md"
+        }
+      ]
+    },
+
+    "homepageURL": "https://github.com/USGS-Astrogeology/knoten/releases/tag/0.1.0",
+    "downloadURL": "https://github.com/USGS-Astrogeology/knoten/archive/refs/tags/0.1.0.zip",
```

Comment:
These URLs need to be updated for 0.2.1

Reply:
So we haven't actually released 0.2.1. I could tag a release which would make those paths valid. With no release tagged they currently go nowhere